database/migrations/2024_06_19_010901_create_participants_table.php

```diff
@@ -25,8 +27,8 @@ public function up(): void
             $table->string('home_phone_number', 12)->nullable();
             $table->string('work_phone_number', 12)->nullable();
             $table->string('alt_contact_number', 12)->nullable();
-            $table->enum('marital_status', ['single', 'married', 'divorced', 'widowed']);
-            $table->enum('ethnicity', ['white', 'african_american', 'native_american', 'asian', 'pacific_islander', 'hispanic', 'no_answer']);
+            $table->enum('marital_status', MaritalStatus::values());
+            $table->enum('ethnicity', Ethnicity::values());
```

Comment:
Maybe add a comment here about replacing this with static values once we've applied our migrations to prod